extension/jaegerremotesampling/extension.go

```diff
@@ -48,14 +48,15 @@ func (jrse *jrsExtension) Start(ctx context.Context, host component.Host) error
 			StrategiesFile: jrse.cfg.Source.File,
 			ReloadInterval: jrse.cfg.Source.ReloadInterval,
 		}
-		ss, err := static.NewStrategyStore(opts, jrse.telemetry.Logger)
-		if err != nil {
-			return fmt.Errorf("failed to create the local file strategy store: %w", err)
-		}
-
-		// there's a Close function on the concrete type, which is not visible to us...
-		// how can we close it then?
-		jrse.samplingStore = ss
+		_ = opts
```

Comment:
Please don't commit commented out code, it makes it hard to maintain the code base.